packages/odyssey-react-mui/src/Button.tsx

```diff
@@ -104,14 +110,22 @@ export type ButtonProps = {
 ) &
   SeleniumProps;
 
+export type ButtonContextValue = {
+  isFullWidth?: boolean;
+};
+
+export const ButtonContext = createContext<ButtonContextValue>({
+  isFullWidth: undefined,
+});
```

Comment:
We can default this to `false`:

```suggestion
export type ButtonContextValue = {
  isFullWidth?: boolean;
};

export const ButtonContext = createContext<ButtonContextValue>({
  isFullWidth: false,
});
```